src/bp-templates/bp-nouveau/buddypress-functions.php

```diff
@@ -339,7 +339,12 @@ public function platform_login_scripts() {
 	 *
 	 * @since BuddyPress 3.0.0
 	 */
-	public function enqueue_styles() {
+	public function enqueue_styles( $page ) {
```

Comment:
@shohelbuddyboss `bp_enqueue_scripts` hook doesn't have `$page` param? is it going to work?

May be you will need global variable to check that? some thing like $current_screen. https://deluxeblogtips.com/get-admin-screen-information/

Check for network screen as well.